business-central-parent/business-monitoring-webapp/src/main/resources/META-INF/services/org.uberfire.java.nio.file.spi.FileSystemProvider

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 #
 
+org.uberfire.java.nio.fs.k8s.K8SFileSystemProvider    # can be set as default at runtime by system property: org.kie.workbench.controller.openshift.enabled
```

Comment:
@adrielparedes When enabling k8sfs, we will still need to keep jGit provider listed as the first as per this [implementation](https://github.com/kiegroup/appformer/blob/0d8c104f2bdb182ca2768eaa5d3a18e57f6f8d1a/uberfire-nio2-backport/uberfire-nio2-api/src/main/java/org/uberfire/java/nio/file/api/FileSystemProviders.java#L79-L87). Second, appreciated if you could update the comments a bit as the name  of the system property as being changes to org.kie.**server**.controller.openshift.enabled, Thanks!